mithril-aggregator/src/signed_entity_service.rs

```diff
@@ -120,9 +120,23 @@ impl SignedEntityService for MithrilSignedEntityService {
             "certificate_hash" => &certificate.hash
         );
 
-        let artifact = self
-            .compute_artifact(signed_entity_type.clone(), certificate)
-            .await?;
+        let mut remaining_retries = 3;
+        let artifact = loop {
+            remaining_retries -= 1;
+
+            match self
+                .compute_artifact(signed_entity_type.clone(), certificate)
+                .await
+            {
+                Err(e) => {
+                    if remaining_retries == 0 {
+                        break Err(e);
+                    }
+                }
+                Ok(a) => break Ok(a),
```

Comment:
For readability can you write `artifact` instead of `a` (and `err`/`error` instead of `e`) ?